gateway/apps/main/src/main.rs

```diff
@@ -14,16 +16,30 @@ use tracing::instrument;
 async fn main() -> anyhow::Result<()> {
     let _logger_guard = init_logger();
 
-    let app_config = ServerConfig::init_config(ConfigVariant::init())?;
-    let postgres_creds = PostgresDbCredentials::from_db_url()?;
-    let db_pool = PostgresRepo::from_config(postgres_creds).await?.into_shared();
-    let app = gateway_server::init::create_app(db_pool).await?;
+    let config_path = std::env::var("CONFIG_PATH").unwrap();
+
+    let app_config = ServerConfig::init_config(ConfigVariant::OnlyOneFilepath(config_path))?;
+    tracing::debug!("App config: {:?}", app_config);
+
+    let frost_aggregator = create_aggregator_from_config(app_config.clone());
+
+    let postgres_url = std::env::var("DATABASE_URL").unwrap();
```

Comment:
```suggestion
    let postgres_creds = PostgresDbCredentials::from_db_url();
```
it takes from DATABASE_URL env
https://github.com/distributed-lab/runes-spark-bridge/blob/895d75383baeb9aac382d5a5932069c15d2c35b5/crates/persistent_storage/src/config.rs#L24C23-L24C29

Reply:
fixed in https://github.com/distributed-lab/runes-spark-bridge/pull/21/commits/42cfe0eac13a639be1c7ac0beabe56c7ddef6e76